dev/fattest.simplicity/src/com/ibm/websphere/simplicity/config/ServerConfiguration.java

```diff
@@ -56,6 +56,12 @@
     @XmlElement(name = "bell")
     private ConfigElementList<Bell> bells;
 
+    @XmlElement(name = "cdi")
+    private ConfigElementList<Cdi> cdi;
+
+    @XmlElement(name = "cdi12")
+    private ConfigElementList<Cdi12> cdi12;
+
```

Comment:
Can only actually have one of these each but I had to make it a list in order to be able to properly remove it (clear the list) during tests